pkgs/build-support/writers/scripts.nix

```diff
@@ -605,8 +774,7 @@ rec {
   writePython3 = makePythonWriter pkgs.python3 pkgs.python3Packages buildPackages.python3Packages;
 
   # writePython3Bin takes the same arguments as writePython3 but outputs a directory (like writeScriptBin)
-  writePython3Bin = name:
-    writePython3 "/bin/${name}";
+  writePython3Bin = name: writePython3 "/bin/${name}";
```

Comment:
In future please do reformatting before you apply other changes such as adding doc strings for easier code review.

Reply:
Yes sorry for the confusion. I had a failing CI Check that told me to Format with nixfmt in another PR and assumed that this is required for all Files now. Seems i need to follow the rfc more closely or wait for the CI to fail.